app/controllers/v1/tiles_controller.rb

```diff
@@ -0,0 +1,40 @@
+class V1::TilesController < ApplicationController
+  def create
+    tile = Tile.new(create_params)
```

Comment:
Only people on the project's team can create tiles, but I don't have the project_people table setup yet. Do you think it's okay to hold off on permissions for this until I have that in?

Here's what I was thinking for it: ![](https://s3.amazonaws.com/screen-caps/Nrx3c.png)